CHANGES.rst

```diff
@@ -118,11 +232,96 @@ New & Improved
 - Optimized request cookie parsing. It is now roughly an order of magnitude
   faster.
 - ``append_header()`` now supports appending raw Set-Cookie header values.
+- Multiple routes can now be added for the same resource instance using a
+  suffix to distinguish the set of responders that should be used. In this way,
+  multiple closely-related routes can be mapped to the same resource while
+  preserving readability and consistency.
+- The ``falcon.media.validators.jsonschema.validate()`` decorator now
+  supports both request and response validation.
+- A static route can now be configured to return the data from a default file
+  when the requested file path is not found.
+- The ordering of the parameters passed to custom error handlers was adjusted
+  to be more intuitive and consistent with the rest of the framework::
+
+    # Before
+    def handle_error(ex, req, resp, params):
+      pass
+
+    # Falcon 2.0
+    def handle_error(req, resp, ex, params):
+      pass
+
+- All error classes now accept a `headers` keyword argument for customizing
+  response headers.
+- A new method, ``get_param_as_float()``, was added to the
+  ``Request`` class.
+- A new method, ``has_param()``, was added to the
+  ``Request`` class.
+- A new property, ``content_length``, was added to the
+  ``Response`` class. Either ``Response.set_stream()`` or
+  ``Response.content_length`` should be used going forward, as
+  ``stream_len`` is now deprecated.
+- All ``get_param_*()`` methods of the ``Request`` class now accept a
+  `default` argument.
+- When setting the ``Response.etag`` header property, the value will
```

Comment:
Theoretically, this could be interpreted as a breaking change and should be listed there too, in case anyone was, intentionally or not, relying on the previous behaviour to set non-conformant values.
But OTOH from the RFC perspective, it just a bug fix.

Reply:
Mmm, I'll add it to breaking changes just in case.

Reply:
(actually, this should be under fixes in the first place)